addon/-private/is-rfc268-test.js

```diff
@@ -0,0 +1,37 @@
+import { getContext, visit } from './compatibility';
+
+export default function isRfc268Test() {
+  // `getContext()` returns:
+  //  - falsey, if @ember/test-helpers is not available (stubbed in
+  //    compatibility.js)
+  //  - falsey, if @ember/test-helpers is available but none of the
+  //    `ember-qunit` setupTest() methods has been called (e.g.,
+  //    `setupRenderingTest()`)
+  //  - truthy, if @ember/test-helpers is available and one of the `ember-qunit`
+  //    setupTest() methods has been called.
+  //
+  // Note that if `page.setContext(this)` has been called, we'll never get here
+  // and will just be running with the integration context (even if the test is
+  // an RFC268 test).
+  let hasValidTestContext = Boolean(getContext());
+  if (!hasValidTestContext) {
+    return false;
+  }
+
+  // There are a few versions of `@ember/test-helpers` that have support for
+  // `ember-qunit`'s `setupRenderingTest()` method, but do not have the DOM
+  // helpers (`click`, `fillIn`, etc.) that the RFC268 execution context uses.
+  // `visit` was the last helper to be added to `@ember/test-helpers`, so we
+  // check for it, and if we can't find it, we can't use the RFC268 execution
+  // context, so we throw an exception.
+  let hasExpectedTestHelpers = Boolean(visit);
+  if (!hasExpectedTestHelpers) {
+    throw new Error([
+      'You are trying to use ember-cli-page-object with RFC232/RFC268 support',
+      '(setupRenderingContext()/setupApplicationContext()) which requires at',
+      'least ember-qunit@3.2.0 or ember-mocha@0.13.0-beta.3.'
```

Comment:
FWIW, that version of `ember-mocha` does *not*  support RFC268-based tests. It might ship with an appropriate version of `@ember/test-helpers`, but it does not yet provide the `setupRenderingTest()/setupApplicationTest()` functions to setup such tests, as `ember-qunit` does...

I know that for sure, because my [PR](https://github.com/emberjs/ember-mocha/pull/190) for adding those just got merged today 😉

Reply:
Heh, okay, should I optimistically change the version to `0.13.2` then?

Reply:
Hm, not sure what version that would be... @Turbo87 any thoughts?

Reply:
it's actually worse than that. even with `ember-qunit@3.2.0` and `visit` available you can't be sure that you can actually use it, because you might be dealing with a testsuite that still uses the old testing APIs and those don't work with the new `visit()` function

Reply:
@Turbo87 this check occures [only](https://github.com/san650/ember-cli-page-object/pull/357/files/464af458da5d7f45582f21c02732d9c0cf334987#diff-af2391cd4be4b040e0c0eddf7e2d444dR16) when we use the new testing API. I think the question here is which minimal versions of `ember-qunit` `ember-mocha` do provide a working `visit()` helper?

Reply:
`visit` was added in `@ember/test-helpers@0.7.9`, which was first required by `ember-qunit@3.2.0`, which is why the error references that version of `ember-qunit`.

@ro0gr is correct -- the question is when `ember-mocha` will support it. As far as I understand, @simonihmig's PR to add full support was just merged, so will that be released as `0.13.2`?